### ISSUE TITLE
Add `CONTRIBUTING.md`

### ISSUE THREAD

**Issue by Snailedlt** [janosh/svelte-multiselect#165]
Would be nice to know what the suggested workflow for contributing is, so that we can more easily make contributions :)

**Comment by Snailedlt** [comment]
I guess so, yes. The whole `Want to contribute?` section can be moved into its own `CONTRIBUTING.md` document. Then in addition to that I'd like to know the following:
Do you have any rules/guidelines for commits, PRs and issues?
Is there any other way we can help out?

**Comment by Snailedlt** [comment]
Would be nice to also get some PR and Issue templates. Different ones for reporting/fixing bugs, and suggesting/implementing PRs would be neat.
Imo a project I maintain does this quite well: https://github.com/devicons/devicon

These templates should then be explained in `CONTRIBUTING.md`

**Comment by Snailedlt** [comment]
Info about the PR checks could also be listed

**Comment by Snailedlt** [comment]
TLDR;
`CONTRIBUTING.md` should contain everything you need to know in order to contribute to the project.

Add the following to `CONTRIBUTING.md`
1. Overview on how we can contribute (report bugs, update documentation, suggest features, implement features, fix bugs, sponsor the project)
2. Rules/guidelines for creating issues (mention templates if you have any)
3. Step by step guide on how to create a PR (from making a branch, to making the PR)
4. List of PR checks (and other github workflows) with explanations

**Comment by janosh** [comment]
Cool, didn't know about `devicon`. Very nice project! [Just refactored](https://github.com/janosh/svelte-multiselect/pull/169/commits/b766f8b32d9e97ee3e046b6e5ed7ae3b1915559e) the `LanguageSlot.svelte` to use it.

**Comment by Snailedlt** [comment]
Awesome!
This makes the project much easier to contribute to, great work!
